### DISCUSSION TITLE
Does not launch on Windows 11 Pro

### DISCUSSION THREAD

**Discussion by DanielAmaya14** [iterate-ch/cyberduck#14858] [Q&A]
**Describe the bug**
I dowloaded cyberduck from the official page, using the last version Version 8.5.9, but the app never launchs when I started it

**To Reproduce**
Only try to launch the app 

**Desktop (please complete the following information):**
Windows 11 pro Lenovo

**Log Files**
This is the crash error reported on windows 
[Report.wer.txt](https://github.com/iterate-ch/cyberduck/files/11523655/Report.wer.txt)

I already try to delete and install again from the website and the microsoft store

**Comment by AliveDevil** [comment]
Please perform following steps, to ensure, we get as most details as possible:
- Download Cyberduck from the website
- Rename the folder "%AppData%\Cyberduck" to something different (Cyberduck.bak)
- Open Event Viewer, navigate to Windows Logs -> Application
- Start Cyberduck
- Once you determined it crashed, please go back to Event Viewer, at the top there should be a "New Logs"-link, click to get the latest events
- Select the first pair of Errors with a source of ".NET Runtime", use "Save selected events", and attach the saved file here (you may need to zip the file, due to Github policy)
- Now navigate into `%AppData%\Cyberduck` and attach, if available, the folder CrashReporter and the cyberduck.log-file (if exists)

Instead of attaching the files on Github, you may send them attached to support@cyberduck.io.

If the issue went away after renaming the Cyberduck-folder then some previous connection may have created a faulty state that you can work around by copying one file after another from the .bak-directory to the newly created Cyberduck-folder, verifying after each file that Cyberduck still starts.
